src/imitation/scripts/data_collect.py

```diff
@@ -24,6 +26,7 @@ def rollouts_and_policy(
   parallel: bool = False,
   normalize: bool = True,
   make_blank_policy_kwargs: dict = {},
+  discrim_net_airl_path: Optional[str] = None,
```

Comment:
I'd like to save this for later, (unless this seems pressing) because there's other imitation things I want to quickly get to. I can make an enhancement issue for this.

Reply:
To be fair, this wouldn't take that long...

Reply:
If you can do it in less than 1.5 hours I'd implement it, otherwise probably not worth it. Agree with avoiding unnecessary complexity, but it's also nice to have consistent interfaces: it's a bit odd to special-case things for a particular algorithm. This also lets us do head-to-head comparisons: AIRL *claims* it learns a more general reward, but it's hard to test that if we don't have other algorithms to compare to.